v7/import_blogger/import_blogger.py

```diff
@@ -191,12 +194,16 @@ def import_item(self, item, out_folder=None):
             # If no content is found, no files are written.
             content = self.transform_content(content)
 
-            self.write_metadata(os.path.join(self.output_folder, out_folder,
-                                             slug + '.meta'),
-                                title, slug, post_date, description, tags)
-            self.write_content(
-                os.path.join(self.output_folder, out_folder, slug + '.html'),
-                content)
+            link_fragments = link_path.split('/')
+            if len(link_fragments) == 4:
```

Comment:
Detecting drafts does not depend on this as far as I can see. You need to write the correct path to the URL map, for example.